Dockerfile

```diff
@@ -8,4 +8,4 @@ FROM alpine:3.14
 
 COPY --from=builder /rdme/exe /exe
 
-ENTRYPOINT ["sh", "-c", "/exe/rdme $INPUT_RDME"]
+ENTRYPOINT ["/exe/rdme"]
```

Comment:
Not only this this a lot cleaner but we can now use good ol' `docker run` arguments, but GitHub recommends[^1] accessing inputs via arguments and not environmental variables [in their own docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs):

<img width="732" alt="CleanShot 2023-04-04 at 18 07 29@2x" src="https://user-images.githubusercontent.com/8854718/229941839-98e91a85-0e22-4d15-84ff-e4cd5a3d4e33.png">

[^1]: They technically imply that it's impossible to access the `INPUT_` environmental variables from the Docker container action, but that's definitely wrong — `rdme@8.6.0`  wouldn't be working at all in that case lol. The takeaway here is that we should be passing inputs via arguments and not environmental variables.